netlify.toml

```diff
@@ -7,4 +7,5 @@
 [[headers]]
   for = "/*"
   [headers.values]
-    X-Frame-Options = "DENY"
\ No newline at end of file
+    X-Frame-Options = "DENY"
```

Comment:
I don't think we need this file at all anymore

Reply:
Turns out we're still using Netlify for prod 😬

Reply:
Should we move the app away from Netlify to secure it further?

Reply:
I don't think it's safe to say it would make it more secure. It's just an effort we're making to consolidate into Vercel. Both hosts have the iframe blocking headers (after this PR, only Netlify does in prod at this moment).